_includes/install-script.md

```diff
@@ -2,54 +2,84 @@
 
 ### Install using the convenience script
 
-Docker provides convenience scripts at [get.docker.com](https://get.docker.com/)
-and [test.docker.com](https://test.docker.com/) for installing edge and
-testing versions of Docker Engine - Community into development environments quickly and
-non-interactively. The source code for the scripts is in the
-[`docker-install` repository](https://github.com/docker/docker-install).
-**Using these scripts is not recommended for production
-environments**, and you should understand the potential risks before you use
-them:
-
-- The scripts require `root` or `sudo` privileges to run. Therefore,
-  you should carefully examine and audit the scripts before running them.
-- The scripts attempt to detect your Linux distribution and version and
-  configure your package management system for you. In addition, the scripts do
-  not allow you to customize any installation parameters. This may lead to an
-  unsupported configuration, either from Docker's point of view or from your own
-  organization's guidelines and standards.
-- The scripts install all dependencies and recommendations of the package
-  manager without asking for confirmation. This may install a large number of
-  packages, depending on the current configuration of your host machine.
-- The script does not provide options to specify which version of Docker to install,
-  and installs the latest version that is released in the "edge" channel.
-- Do not use the convenience script if Docker has already been installed on the
-  host machine using another mechanism.
-
-This example uses the script at [get.docker.com](https://get.docker.com/) to
-install the latest release of Docker Engine - Community on Linux. To install the latest
-testing version, use [test.docker.com](https://test.docker.com/) instead. In
-each of the commands below, replace each occurrence of `get` with `test`.
-
-> **Warning**:
->
-Always examine scripts downloaded from the internet before
-> running them locally.
+Docker provides a convenience script at [get.docker.com](https://get.docker.com/)
+to install Docker into development environments quickly and non-interactively.
+The convenience script is not recommended for production environments, but can be
```

Comment:
Can we have a link for the "recommended" way

Reply:
Added:

```markdown
Also refer to the [install using the repository](#install-using-the-repository)
steps to learn about installation steps to install using the package repository.
```